src/sentry/middleware/integrations/parsers/base.py

```diff
@@ -114,12 +114,14 @@ def get_response_from_outbox_creation(self, regions: Sequence[Region]):
         Used to create outboxes for provided regions to handle the webhooks asynchronously.
         Responds to the webhook provider with a 202 Accepted status.
         """
-        for outbox in ControlOutbox.for_webhook_update(
-            webhook_identifier=self.webhook_identifier,
-            region_names=[region.name for region in regions],
-            request=self.request,
-        ):
-            outbox.save()
+        if len(regions) > 0:
```

Comment:
Do we end up with 0 regions when a webhook for a disconnected integration are received? Should this be a condition that we log or track metrics for?

Reply:
It's actually just error catching in case we ever run this function with an empty list. If there are no regions, the parsers themselves either log or error on that, this just avoids any issues at runtime.